distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -66,8 +69,8 @@ fill length ~item =
                       @Tail_Call do_build start+1 stop
               do_build 1 10
               builder.to_vector
-new_builder : Builder
-new_builder = Builder.new
+new_builder : Integer -> Builder
+new_builder (capacity=1) = Builder.new capacity
```

Comment:
The example above needs to be adjusted?

Reply:
no as capacity is optional. 

But more efficient so amended.